src/wp-includes/ms-blogs.php

```diff
@@ -575,7 +575,7 @@ function switch_to_blog( $new_blog_id, $deprecated = null ) {
 				);
 			}
 
-			wp_cache_add_non_persistent_groups( array( 'counts', 'plugins', 'theme_json' ) );
```

Comment:
I think this change goes beyond what we should possibly revert here. Why should we revert adding the non-persistent cache group?

Reply:
WHAT? That's not desirable. Darn it. Will fix.

Reply:
It only removed `theme_json'` from each of these. https://core.trac.wordpress.org/changeset/55092

Reply:
@hellofromtonya I understand, I meant the addition of `theme_json` here was not responsible for the fatal error, so I'm not sure we should remove it. But anyway, let's follow up on a new PR :)